src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx

```diff
@@ -41,7 +43,12 @@ export const JobDetailAbout: React.FC<JobDetailAboutProps> = ({ isUser = true, s
   const [openApply, setOpenApply] = useState(false);
   const [openExternalApply, setOpenExternalApply] = useState(false);
   const [openAlert, setOpenAlert] = useState(false);
-  const url = window.location.href;
+  let url = window.location.href;
```

Comment:
please use `useLocation()` instead of using Web API.

Reply:
ah I already mentioned a solution on other branch for @bastantebom still this one, useLocation and window.location both  would be wrong when it come up to moblie app